msg/position_setpoint.msg

```diff
@@ -40,6 +40,7 @@ int8 loiter_direction		# loiter direction: 1 = CW, -1 = CCW
 float32 acceptance_radius   # navigation acceptance_radius if we're doing waypoint navigation
 
 float32 cruising_speed		# the generally desired cruising speed (not a hard constraint)
-float32 cruising_throttle	# the generally desired cruising throttle (not a hard constraint)
+bool gliding_enabled		# commands the vehicle to glide if the capability is available (fixed wing only)
+float32 cruising_throttle	# the generally desired cruising throttle (not a hard constraint), only has an effect for rover
```

Comment:
With your latest changes (i.e. doing gliding with dedicated flag), cruising_throttle is not used anymore in fixed-wing position controller. 
I see that it's though still used for Rover and think there it could make sense to keep it. @Jaeyoung-Lim I guess that's the case?
If we can remove it all together - great, this removes also quite some handling in Navigator - if not: let's make it clear that it's a Rover-only feature (at least in the comment here).